README.md

```diff
@@ -45,35 +48,35 @@ cd ovsrpro
 git checkout <tag>		# where tag is, for example 17.02.1
 mkdir ovsrpro-build
 cd ovsrpro-build
-cmake ../ovsrpro -DXP_STEP=build
+cmake ../ovsrpro -DXP\_STEP=build
```

Comment:
If you're making changes it would be good to fix the HDF5 download URL, apparently they changed the layout recently, just discovered that yesterday helping @swflb get the latest to build.  The current URL is http://www.hdfgroup.org/ftp/HDF5/prev-releases/hdf5-${HDF5_VER}/src/CMake-hdf5-${HDF5_VER}.tar.gz, but there needs to be an extra 'hdf5-1.8' in it so the populated url for our current version would be http://www.hdfgroup.org/ftp/HDF5/prev-releases/hdf5-1.8/hdf5-1.8.16/src/CMake-hdf5-1.8.16.tar.gz